rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy

```diff
@@ -370,6 +372,110 @@ class ScheduledExecutionController  extends ControllerBase{
+                )
+        ],
+            responses = @ApiResponse(
+                    responseCode = '200',
+                    description = '''Job Definition Components response.
+
+**Components Fields**
+* `properties`: List of properties for the component
+* `section`: Section of the component
+* `pluginConfig`: Plugin configuration for the component
+* `prefix`: Prefix for the component
+* `messageType`: Message type for the component
+''',
+                    content = @Content(
+                            mediaType = MediaType.APPLICATION_JSON,
+                            schema = @Schema(type = 'object'),
+                            examples = @ExampleObject('''{
+    "job-tags":{
+      "properties":[
+         {
+            "blankIfUnexpandable":true,
+            "defaultValue":null,
+            "description":"Input multiple tags separated by commas. Remove a tag by clicking on it.",
+            "name":"tags",
+            "renderingOptions":{
+               
+            },
+            "required":false,
+            "scope":null,
+            "selectLabels":null,
+            "selectValues":null,
+            "title":"Tags",
+            "type":"String",
+            "validator":null
+         }
+      ],
+      "section":"details",
+      "pluginConfig":{
+         "job-queue":null,
+         "job-tags":null,
+         "Schedules-component":{
+            "schedulesJson":"[\\n    \\n]"
+         },
+         "runner-job-selector":{
+            "runnerSelectorJson":"{\\"runnerFilterType\\":\\"TAG_FILTER_AND\\",\\"filter\\":\\"STG-TOOLS-NEW\\",\\"runnerFilterMode\\":\\"TAGS\\"}"
+         }
+      },
+      "prefix":"jobComponent.job-tags.configMap.",
+      "messageType":"jobComponent.job-tags"
+   }
+}''')
+                    )
+            )
+    )
+    def apiJobDefinitionComponentsValues() {
+        ScheduledExecution scheduledExecution = scheduledExecutionService.getByIDorUUID( params.id )
+
+        if (!scheduledExecution) {
+            return apiService.renderErrorFormat(response, [
+                    status: HttpServletResponse.SC_NOT_FOUND,
+                    code: 'api.error.job.not.found',
+                    args: [params.id],
+                    format: response.format
+            ])
+        }
+
+        def jobComponentValues=rundeckJobDefinitionManager.getJobDefinitionComponentValues(scheduledExecution)
+        def jobComponents = rundeckJobDefinitionManager.getJobDefinitionComponents()
+        Map componentsData= [:]
+        jobComponents.collect { String name, JobDefinitionComponent jobComponent ->
+            if (!jobComponent.inputProperties) {
+                return
+            }
+            def compSection = jobComponent.inputLocation?.section
+            String prefix = RundeckJobDefinitionManager.getFormFieldPrefixForJobComponent(jobComponent.name)
+            String messageType = RundeckJobDefinitionManager.getMessagesTypeForJobComponent(jobComponent.name)
```

Comment:
I think the prefix and messageType may not be necessary. At least prefix I know will not be necessary to include. and can be removed